service/controller/v21/templates/cloudformation/guest/vpc.go

```diff
@@ -23,4 +23,26 @@ const VPC = `{{define "vpc"}}
       Tags:
         - Key: Name
           Value: {{ $v.ClusterID }}
```

Comment:
I completely miss why we need this endpoint. I'd appreciate some more information in the PR desc <3

Reply:
https://github.com/giantswarm/giantswarm/issues/4927

customer requested this feature, it allows access s3 without going over public network